x-pack/plugins/secops/public/components/error_toast/index.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiGlobalToastList, Toast } from '@elastic/eui';
+import { noop } from 'lodash';
+import React from 'react';
+import uuid from 'uuid';
+
+interface Props {
+  toastLifeTimeMs?: number;
+}
+
+interface State {
+  toasts: Toast[];
+}
+
+let showErrorHandler = ({ id = uuid.v4(), title, message }: ShowErrorParams): void => noop();
+export const showError = ({ id = uuid.v4(), title, message }: ShowErrorParams) =>
+  showErrorHandler({ id, title, message });
+
+interface ShowErrorParams {
+  id?: string;
+  title: string;
+  message: string;
+}
+
+export class ErrorToast extends React.PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      toasts: [],
+    };
+    showErrorHandler = this.showError;
+  }
+
+  public showError = ({ id = uuid.v4(), title, message }: ShowErrorParams) => {
+    const toast: Toast = {
+      id,
+      title,
+      color: 'danger',
+      iconType: 'alert',
+      text: <p>{message}</p>,
+    };
+
+    this.setState({
+      toasts: this.state.toasts.concat(toast),
```

Comment:
This is fine for now, but as per the [EUI Guidelines](https://elastic.github.io/eui/#/guidelines/toasts) we should only ever display one toast at a time. I'll open an issue to track this one as there's probably a bit more thought that needs to go into how we're handling toasts.

Reply:
Thanks. Saw this as well, `By default, a toast stays on the screen 10 seconds.`. I should change my default timeout to be 10 seconds.

I don't think I'm spamming them unless for some reason they're doing user generated errors more than 1 at a time or their network goes completely belly up and then they try to refresh their date time :-)

At that point...Yea, they get a lot of 🍞 toasts!